core/src/main/java/io/questdb/PropertyKey.java

```diff
@@ -0,0 +1,350 @@
+    LINE_TCP_NET_CONNECTION_HINT("line.tcp.net.connection.hint"),
+    LINE_TCP_NET_BIND_TO("line.tcp.net.bind.to"),
+    LINE_TCP_NET_IDLE_TIMEOUT("line.tcp.net.idle.timeout"),
+    LINE_TCP_NET_CONNECTION_TIMEOUT("line.tcp.net.connection.timeout"),
+    LINE_TCP_NET_QUEUED_TIMEOUT("line.tcp.net.queued.timeout"),
+    LINE_TCP_NET_CONNECTION_QUEUE_TIMEOUT("line.tcp.net.connection.queue.timeout"),
+    LINE_TCP_NET_RECV_BUF_SIZE("line.tcp.net.recv.buf.size"),
+    LINE_TCP_NET_CONNECTION_RCVBUF("line.tcp.net.connection.rcvbuf"),
+    LINE_TCP_CONNECTION_POOL_CAPACITY("line.tcp.connection.pool.capacity"),
+    LINE_TCP_TIMESTAMP("line.tcp.timestamp"),
+    LINE_TCP_MSG_BUFFER_SIZE("line.tcp.msg.buffer.size"),
+    LINE_TCP_MAX_MEASUREMENT_SIZE("line.tcp.max.measurement.size"),
+    LINE_TCP_WRITER_QUEUE_CAPACITY("line.tcp.writer.queue.capacity"),
+    LINE_TCP_WRITER_WORKER_COUNT("line.tcp.writer.worker.count"),
+    LINE_TCP_WRITER_WORKER_AFFINITY("line.tcp.writer.worker.affinity"),
+    LINE_TCP_WRITER_HALT_ON_ERROR("line.tcp.writer.halt.on.error"),
+    LINE_TCP_WRITER_WORKER_YIELD_THRESHOLD("line.tcp.writer.worker.yield.threshold"),
+    LINE_TCP_WRITER_WORKER_SLEEP_THRESHOLD("line.tcp.writer.worker.sleep.threshold"),
+    LINE_TCP_SYMBOL_CACHE_WAIT_US_BEFORE_RELOAD("line.tcp.symbol.cache.wait.us.before.reload"),
+    LINE_TCP_IO_WORKER_COUNT("line.tcp.io.worker.count"),
+    LINE_TCP_IO_WORKER_AFFINITY("line.tcp.io.worker.affinity"),
+    LINE_TCP_IO_HALT_ON_ERROR("line.tcp.io.halt.on.error"),
+    LINE_TCP_IO_WORKER_YIELD_THRESHOLD("line.tcp.io.worker.yield.threshold"),
+    LINE_TCP_IO_WORKER_SLEEP_THRESHOLD("line.tcp.io.worker.sleep.threshold"),
+    LINE_TCP_MAINTENANCE_JOB_INTERVAL("line.tcp.maintenance.job.interval"),
+    LINE_TCP_COMMIT_INTERVAL_FRACTION("line.tcp.commit.interval.fraction"),
+    LINE_TCP_COMMIT_INTERVAL_DEFAULT("line.tcp.commit.interval.default"),
+    LINE_TCP_AUTH_DB_PATH("line.tcp.auth.db.path"),
+    LINE_TCP_DEFAULT_PARTITION_BY("line.tcp.default.partition.by"),
+    LINE_DEFAULT_PARTITION_BY("line.default.partition.by"),
+    LINE_TCP_MIN_IDLE_MS_BEFORE_WRITER_RELEASE("line.tcp.min.idle.ms.before.writer.release"),
+    LINE_TCP_DISCONNECT_ON_ERROR("line.tcp.disconnect.on.error"),
+    LINE_TCP_UNDOCUMENTED_STRING_TO_CHAR_CAST_ALLOWED("line.tcp.undocumented.string.to.char.cast.allowed"),
+    LINE_TCP_UNDOCUMENTED_SYMBOL_AS_FIELD_SUPPORTED("line.tcp.undocumented.symbol.as.field.supported"),
+    LINE_TCP_UNDOCUMENTED_STRING_AS_TAG_SUPPORTED("line.tcp.undocumented.string.as.tag.supported"),
+    METRICS_ENABLED("metrics.enabled"),
+    PG_ENABLED("pg.enabled"),
+    PG_NET_CONNECTION_HINT("pg.net.connection.hint"),
+    PG_NET_BIND_TO("pg.net.bind.to"),
+    PG_NET_IDLE_TIMEOUT("pg.net.idle.timeout"),
+    PG_NET_CONNECTION_TIMEOUT("pg.net.connection.timeout"),
+    PG_NET_CONNECTION_QUEUE_TIMEOUT("pg.net.connection.queue.timeout"),
+    pg_NET_RECV_BUF_SIZE("pg.net.recv.buf.size"),
```

Comment:
`pg_NET_RECV_BUF_SIZE` -> `PG_NET_RECV_BUF_SIZE`

Reply:
Will fix this asap 😄